docs/appendices/release-notes/5.7.2.rst

```diff
@@ -52,6 +52,15 @@ Security Fixes
 Fixes
 =====
 
+- Fixed issue that threw an ``ArrayIndexOutOfBoundsException`` instead of a user
+  friendly error message, instead of, when the ``WHERE``` clause of a query
+  uses all columns of a :ref:`PRIMARY KEY <constraints-primary-key>`, uses
+  parameters, and binds less actual values than the required, e.g.::
```

Comment:
```suggestion

- Fixed an issue leading to a ``ArrayIndexOutOfBoundsException``  instead of a user
  friendly error message when the ``WHERE``` clause of a query
  uses all columns of a :ref:`PRIMARY KEY <constraints-primary-key>`, uses
  parameters, and binds less actual values than the required, e.g.::
```

Reply:
We have another fix entry with ArrayIndexOutOfBoundsException, let's make phrasing consistent. Also, duplicate "instead of".

Reply:
thx!